workflow/README.md

```diff
@@ -0,0 +1,47 @@
+# TPS Pipeline
+
+**TPS workflow overview**
+
+![Alt Text](https://github.com/ajshedivy/tps/blob/visualization_v2/workflow/TPS_workflow.png)
+
+The TPS/workflow module utilizes the TPS repository structure in order to run the TPS software end-to-end in order to produce Cytoscape sessions. 
+
+## Documentation
+
+- `TPS.sh`
+  - Runs TPS software 
+  - sets up and initializes workspace environment `py2cyto_envirnment`
+  - calls driver script `main.py`
+
+- `main.py`
+  - redirects TPS output files
+  - calls utility functions for processing network annotations
+  - calls visualization utilities
+  
+- `visualization_utilities.py`
+  - utility functions used for creating cyREST client 
+  - import ooutput groah files, style and annotation into Cytoscape
+  - TODO : search machine for Cytoscape executable
+  
+- `table_refactor.py`
+  - helper methods of reformatting annotations data
+
+
```

Comment:
Add `table_generation.py` to this list.

If we want to include the TPS output files as examples for these scripts, we may want to move them and rename them to make it more clear what they are.  Or we could regenerate them every time.